packages/clerk-js/src/v4/customizables/useContainerMetadata.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+type ContainerMetadataProps = {
+  flow: 'signIn' | 'signUp';
+  page: 'start' | 'emailOtp' | 'phoneOtp';
+  blockingAction?: boolean;
+};
+
+type ContainerMetadataContextValue = ContainerMetadataProps;
+const ContainerMetadataContext = React.createContext<{ value: ContainerMetadataContextValue } | undefined>(undefined);
```

Comment:
nit: why is this type have to be reassigned? 
couldn't it be just this?

```
type ContainerMetadataContextValue = {
  flow: 'signIn' | 'signUp';
  page: 'start' | 'emailOtp' | 'phoneOtp';
  blockingAction?: boolean;
};
```

Reply:
Thanks John, this has been fixed here #268 
Please do nit pick :)